$OpenBSD: patch-icinga-app_icinga_cpp,v 1.1 2014/11/22 00:14:05 sthen Exp $

XXX Work around random crashes at program startup. Typically these
have a corrupt stack, but occasionally the backtrace includes
Application::SetResourceLimits or related functions.

--- icinga-app/icinga.cpp.orig	Fri Nov 21 23:49:09 2014
+++ icinga-app/icinga.cpp	Fri Nov 21 23:49:30 2014
@@ -103,8 +103,10 @@ int Main(void)
 
 	Application::SetStartTime(Utility::GetTime());
 
+#if 0
 	if (!autocomplete)
 		Application::SetResourceLimits();
+#endif
 
 	/* Set thread title. */
 	Utility::SetThreadName("Main Thread", false);
